lib/qunit-bdd.js

```diff
@@ -68,6 +68,7 @@
           try {
             fn.call(env);
           } catch (e) {
+            e.stack && console.warn(e.stack);
```

Comment:
The backtrace becomes present, but not clickable:

![image](https://cloud.githubusercontent.com/assets/412371/15723720/a9b8e4a6-27f8-11e6-910f-06c12ccb57cf.png)

For whatever reason, there's a discrepancy between links being clickable if a `console.*()` is used with any other arguments. For example:

```js
try {
  // do stuff
} catch (e) {
  console.warn("Some error happened", e.stack); // Links might not be clickable
  console.warn(e.stack); // Links are clickable
}